### ISSUE TITLE
about the configuration for the run

### ISSUE THREAD

**Issue by Shayne-Pro** [shuaichaochao/HybridHash#2]
I ran 300 epochs with the following configuration, and the best MAP (mean average precision) was only 0.348.
Could you please share the specific configuration you used for the run?
![codetoimg-snippet](https://github.com/shuaichaochao/HybridHash/assets/114326452/343f7dae-6789-4bd0-97cc-baf700356ebe)

**Comment by shuaichaochao** [comment]
The parameter settings are the same as in the file (train.py), have you loaded the pre-trained model weights (jx_nest_base-8bc41011.pth) yet?

**Comment by YinqiChen-DHQ** [comment]
I loaded the pre-trained model weights (jx_nest_base-8bc41011.pth) and also ran 300 epochs(64 batch_size), but the best MAP was only 0.323 in imagenet (64bit).  Could you please share the number of loss that can achieve better results?

**Comment by shuaichaochao** [comment]
> I loaded the pre-trained model weights (jx_nest_base-8bc41011.pth) and also ran 300 epochs(64 batch_size), but the best MAP was only 0.323 in imagenet (64bit). Could you please share the number of loss that can achieve better results?

I've uploaded the training log for your reference.

**Comment by shuaichaochao** [comment]
> I loaded the pre-trained model weights (jx_nest_base-8bc41011.pth) and also ran 300 epochs(64 batch_size), but the best MAP was only 0.323 in imagenet (64bit). Could you please share the number of loss that can achieve better results?

The previous (HybridHash.py) model code was set up as a no-interaction module, resulting in very low MAP results, which has been corrected. The model code has been set to add the interaction module.

**Comment by Shayne-Pro** [comment]
I appreciate your response. Upon re-executing the code, I was able to successfully replicate the standard outcomes.
